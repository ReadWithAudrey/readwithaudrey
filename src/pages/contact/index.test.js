import React from 'react'
import renderer from 'react-test-renderer'
import ContactUs from './index.js'

describe('ContactUs page snapshot test', () =>
  it('renders correctly', () => {
    // console.log(Thankyou)
    const tree = renderer.create(<ContactUs />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
