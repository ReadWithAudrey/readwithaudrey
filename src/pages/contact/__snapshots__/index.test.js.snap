// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs page snapshot test renders correctly 1`] = `
Array [
  <header
    className="sc-bdVaJa flex items-center justify-between ph3 pv2 montserrat white bg-green w-100 f3 gzTCKs"
  >
    <div
      className="flex items-center"
      id="header-left"
    >
      <Link
        to="/"
      >
        <img
          alt="book icon"
          className="h2 ph2"
          src="test-file-stub"
        />
      </Link>
      <Link
        to="/"
      >
        Audrey
      </Link>
    </div>
    <nav
      className="sc-bwzfXH w-65 w-40-l montserrat f4 gQAWUv"
    >
      <section
        className="sc-htpNat montserrat bRYTxq"
      >
        <Link
          to="/story/"
        >
          Why?
        </Link>
        <Link
          to="/faq/"
        >
          FAQ
        </Link>
        <Link
          to="/contact/"
        >
          Contact
        </Link>
        <Link
          className="mv2 white br2 ba bw1 b--white"
          to="/form1"
        >
          Join Us
        </Link>
      </section>
      <section
        className="sc-bxivhb igYaY"
      >
        <a
          className="dtc fr tr v-mid dn-ns p0-ns link dim white"
          onClick={[Function]}
        >
          <span
            className="db w2 mt1 mb2 bw2 pb2 bt bb b--white"
          />
          <span
            className="db w2 mt1 bw2 pb1 bt b--white"
          />
        </a>
      </section>
    </nav>
  </header>,
  <div
    className="page-padding ph4 w-100 w-70-l center"
  >
    <h1
      className="f2 pink tc montserrat mb3 mt4"
    >
      How can we help you?
    </h1>
    <p
      className="sc-kAzzGY montserrat f5 tc lh-copy jdrnSq"
    >
      We would love to hear from you. Your questions and comments are important to us. The best way to reach us is in writing. Please fill out the contact form below:
    </p>
    <form
      action="http://localhost:5000/formPart1"
      method="POST"
    >
      <label
        className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
      >
        Your Name
      </label>
      <input
        className="sc-gzVnrw montserrat mv2 pa2 w-100 ildXbv"
        placeholder="Your Name"
      />
      <label
        className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
      >
        Your Email
      </label>
      <input
        className="sc-gzVnrw montserrat mv2 pa2 w-100 ildXbv"
        placeholder="Your Email"
      />
      <label
        className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
      >
        Your Message
      </label>
      <textarea
        className="sc-chPdSV montserrat mZiEN"
        placeholder="Your Message"
        rows="5"
      />
      <Link
        className="no-underline"
        to="/thankyou"
      >
        <button
          className="sc-gZMcBi bg-pink hover-bg-orange white pa2 br2 w-40 w-30-ns h3 f3 mt4 center gTtpNx"
        >
          Submit
        </button>
      </Link>
      <p
        className="sc-kAzzGY montserrat f5 tc lh-copy jdrnSq"
      >
        Want to get involved and help us do amazing things? We’d be delighted to chat.
      </p>
      <p
        className="sc-kAzzGY montserrat f5 tc lh-copy jdrnSq"
      >
        Fancy a coffee? We float between independent bookshops, coffee houses and libraries in Kings Cross, London, England – around the corner from Platform 9¾.
      </p>
      <p
        className="sc-kAzzGY montserrat f5 tc lh-copy jdrnSq"
      >
        Oh…. almost forgot…. if you’ve got a great book recommendation please please share it with us!
      </p>
    </form>
  </div>,
  <footer
    className="sc-EHOje flex flex-column justify-center items-center montserrat bg-green w-100 bt bw2 b--green jvzENE"
  >
    <span
      className="sc-bZQynM flex justify-around items-center bg-white black w-100  pa2 jrqyAQ"
    >
      © Audrey
      <Link
        className="no-underline black ph1"
        to="/terms/"
      >
        Terms
      </Link>
      <Link
        className="no-underline black ph1"
        to="/privacy/"
      >
        Privacy
      </Link>
      <Link
        className="no-underline black ph1"
        to="/contact/"
      >
        Contact
      </Link>
    </span>
  </footer>,
]
`;
